### ISSUE TITLE
Capistrano is not restarting rescue-pool after deploys

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/newfia#2]
**STEPS TO REPRODUCE**
1) Make sure rescue-pool is started
2) Deploy code using capistrano
3) Check rescue-pool status

*EXPECTED RESULT:* `resque-pool is started`
*ACTUAL RESULT:* `resque-pool is stopped`

**Comment by mark-dce** [comment]
I'm seeing this on the `internal_vm` stage against vagrant right now, but don't see it occurring on the `aws` stage against EC2

**Comment by mark-dce** [comment]
```
INFO [6ab4fcf2] Finished in 0.003 seconds with exit status 0 (successful).
vagrant@ubuntu-14:~/newfia$ sudo service resque-pool status
resque-pool is stopped
vagrant@ubuntu-14:~/newfia$ sudo service resque-pool start
/etc/init.d/resque-pool ↠ starting resque-pool
resque-pool running, pool master started
vagrant@ubuntu-14:~/newfia$ bundle exec cap internal_vm deploy
INFO [e11bc89f] Running /usr/bin/env mkdir -p /tmp/newfia/ as deploy@localhost

. . . ( cap output omitted)

DEBUG [98b5680e] Command: echo "Branch master (at fd0770f) deployed as release 20151215173948 by vagrant" >> /opt/sufia-project/revisions.log
INFO [98b5680e] Finished in 0.002 seconds with exit status 0 (successful).
vagrant@ubuntu-14:~/newfia$ sudo service resque-pool status
resque-pool is stopped
vagrant@ubuntu-14:~/newfia$ 
```

**Comment by acozine** [comment]
Fixed the newfia end of this issue with a0eec79c670ed419a9d9c7bc9e4116abeef5e140
